CHANGES/942.contrib.rst

```diff
@@ -0,0 +1,5 @@
+In the GitHub Actions CI/CD workflow definition,
+the ``Get pip cache dir`` step has been fixed for
+Windows runners by adding ``shell: bash``.
+See `this issue <https://github.com/actions/runner/issues/2224>`_
```

Comment:
> WARNING: Duplicate explicit target name: "this issue"

`this issue` is [not a valid link caption](https://readthedocs.org/projects/multidict/builds/23318806/) since there is already `this issue` in other news fragment 😅

Reply:
You can use a double trailing underscore for anonymous references.

Reply:
Do you want me to update the PR or keep `actions/runner#2224` as the link caption?

Reply:
FWIW some of the changes don't actually need a change note. Notes should be about the high level impact, not low-level internals.